thrift/src/main/thrift/rpc.thrift

```diff
@@ -416,6 +416,12 @@ struct TSDropSchemaTemplateReq {
   2: required string templateName
 }
 
+struct TSOperationSyncWriteReq {
```

Comment:
We should also update "rpc-changelist.md" file

Reply:
Fixed.